### PR TITLE
adding a blog post on how to integrate unlock in a react application

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title!
This was inspired by a request by our friends at Scout!

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [X] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [X] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [X] I have performed a self-review of my own code
- [X] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

**Comment by cellog** [comment]
This is very nice. If you want the post to have traction in today's React world, it really should be rewritten to use hooks. I'm happy to suggest the needed changes if you agree

File: unlock-protocol.com/blog/integratating-unlock-react.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+        )}
+      </header>
+    </div>
+  )
+}
+```
+
+## Conclusion
+
+Unlock's JavaScript API is very simple and will go out of the way to let you implement behaviors that your application requires. This blog only lets members comments, but also shows the status in the bottom bar...
```
Comment (akeem) [inline unlock-protocol.com/blog/integratating-unlock-react.md]:
```suggestion
Unlock's JavaScript API is very simple and will get out of the way to let you implement behaviors that your application requires. This blog only lets members comments, but also shows the status in the bottom bar...
```

**Comment by julien51** [comment]
@cellog that's a great idea for a follow up! I think many many code bases do not use hooks yet, so let's make sure we have a "legacy" version too!

@akeem there's actually an illustration for this post: the react icon :)

**Comment by julien51** [comment]
image

**Pull request merged**